app/scripts/controllers/paymentModal.js

```diff
@@ -326,6 +326,7 @@ angular.module('confRegistrationWebApp')
     /*
     Returns false is payment if payment and refund channel are credit card and current time is less than 24 hours from
     payment time.  If a refund is issued before the payment settles, authorize.net will do a full refund regardless of amount.
+    ***Authorize.net no longer used, but payment is still used so keeping code below for now.***
     */
     $scope.isPartialRefundAvailable = function(payment, refundChannel) {
```

Comment:
Partial refunds exists with TSYS. I don't know that they have the same restrictions as Authorize.NET did. I think they don't.